packages/client/src/ActionsAndReducers/dbMessageTypes/messageTypes_Reducer.ts

```diff
@@ -0,0 +1,32 @@
+import * as ActionConstant from '@serge/config'
+import copyState from '../../Helpers/copyStateHelper'
+import { MessageTypes, MessageTypesActionTypes } from '@serge/custom-types'
+
+const initialState: MessageTypes = {
+  loadingDb: false,
+  isLoading: false,
+  messages: [],
+  messagesByKey: {}
```

Comment:
is this actually `templatesByKey`?  It looks like a list of templates, rather than a list of messages.

Reply:
Hmm I think that `messages` should also be `templates`.

Aah, it's short for _message-types_, which are also called _templates_.  Could we switch to `templates`, to avoid confusion with lists of messages?

Reply:
Here is an example of where we could get confusion:
![image](https://user-images.githubusercontent.com/1108513/127146254-f04ea59b-6ea6-44c0-be67-8d3675a3cd7b.png)